framework/test_app/runners/k8s/k8s_xds_client_runner.py

```diff
@@ -156,6 +156,8 @@ def run(  # pylint: disable=arguments-differ
             server_target=server_target,
             rpc=rpc,
             qps=qps,
+            request_payload_size=271828,
```

Comment:
These should be arguments of the `run` method, and set to 0 by default.

Reply:
Fixed

Reply:
Fixed